buy_together_app/tests/test_login.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from unittest.mock import Mock
+from buy_together_app import views
+
+
+class Test_Login:
+    @pytest.fixture
+    def authenticate_mock(self, monkeypatch):
+        mock = Mock()
```

Comment:
Nice use of mocking. :) check out [create_autospec](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.create_autospec), it is more useful.